### PR TITLE
Skip updating for XP and OSX 10.10

### PR DESCRIPTION
**Pull request by joesis**
For getlantern/lantern-internal#2752

### CODE REVIEW

**Comment by joesis** [comment]
CI timed out because it took so long to download the assets from GitHub.

**Comment by joesis** [comment]
Okay CI finally passed by setting an envvar to skip downloading from GitHub and generate random content instead.

File: server/server.go [hunk 1]
```diff
@@ -16,6 +16,8 @@ import (
 
 // Version 3.6.0
 var v360 = semver.MustParse("3.6.0")
+var lastVersionForWindowsXP = "5.3.9"
+var lastVersionForOSXYosemite = "5.3.9"
```
Comment (myleshorton) [inline server/server.go]: This can actually be 5.4.0 since I build that with the old go...

File: server/server.go [hunk 2]
```diff
@@ -278,7 +296,6 @@ func (u *UpdateServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	w.Header().Set("X-Message-Signature", hex.EncodeToString(messageAuth))
 
-	w.WriteHeader(http.StatusOK)
```
Comment (myleshorton) [inline server/server.go]: It does this by default is that right @joesis?

File: server/server.go (hunk 1 above)
Comment (joesis) [reply server/server.go]: Cool. Will change.

File: server/server.go (hunk 2 above)
Comment (joesis) [reply server/server.go]: Yeah that's the default.

File: server/server.go (hunk 1 above)
Comment (joesis) [reply server/server.go]: Done

**Comment by myleshorton** [comment]
Not sure what's up with CI, but tests are succeeding locally for me...merging!

**Comment by joesis** [comment]
Thanks @myleshorton ! Will deploy and have some test.

**Pull request merged**